Src/BootCamp.Chapter/ComputerFactory.cs

```diff
@@ -4,22 +4,19 @@ namespace BootCamp.Chapter
 {
     public abstract class ComputerFactory
     {
-        protected readonly Body _body;
-        protected readonly Motherboard _motherboard;
-        protected readonly HardDisk _hard;
-        protected readonly Cpu _cpu;
-        protected readonly Gpu _gpu;
-        protected readonly Ram _ram;
+        protected Body _body;
```

Comment:
InstallBody should create and attach body to a computer. You don't need to have references to each piece.

Reply:
As suggested, changed the InstallBody to create and attach body to a computer.